component-library/component-lib/src/lib/form-components/checkbox/checkbox.component.ts

```diff
@@ -97,7 +96,7 @@ export class CheckboxComponent
   errorAria = '';
   errorStubText = '';
   currentStatus: FormControlStatus = 'VALID';
-  currentTouch: boolean = false
+  currentTouch: boolean = false;
 
   constructor(
```

Comment:
Somewhere either in the constructor or in onInit() there should be something like this:

    //set config from individual options, if present
    if (this.formGroup !== this.formGroupEmpty) {
      this.config.formGroup = this.formGroup;
    };
    if (this.id !== '') this.config.id = this.id;
    if (this.size) this.config.size = this.size;
    if (this.label) this.config.label = this.label;
    if (this.desc) this.config.desc = this.desc;
    if (this.hint) this.config.hint = this.hint;
    if (this.placeholder) this.config.placeholder = this.placeholder;
    if (this.required) this.config.required = this.required;
    if (this.options) this.config.options = this.options;
    if (this.errorMessages) this.config.errorMessages = this.errorMessages;
    if (this.topLabel) this.config.topLabel = this.topLabel;
    if (this.disableError) this.config.disableError = this.disableError;

    if (this.config.errorMessages) {
      this.errorIds = this.standAloneFunctions.getErrorIds(
        this.config.formGroup,
        this.config.id,
        this.config.errorMessages
      );
    }